### ISSUE TITLE
A11y: In Time Picker the Tab is used where the arrows Up and Down should be used instead

### ISSUE THREAD

**Issue by polibb** [grafana/grafana#56232]
**What happened**:
In the **quick ranges** moving with arrow up and down is only moving the cursor but not moving through the options. We are using tab to preselect the option. This is not ideal because we need to go through the whole list using the tab to go the options "Absolute time range".

**What you expected to happen**:
Using the arrows for Up and Down on the keyboard should allow the user to go through the list of quick ranges by preselecting each option. The Tab, on the other hand, should allow the user to go to the next section ("Absolute time range" in this case) immediately at any point from the list of quick ranges. 

**How to reproduce it (as minimally and precisely as possible)**:
Open Time Picker in a dashboard. Use the Tab to get to the quick ranges list. Use the arrows Up and Down to see how the cursor moves, but the preselection stays the same.

https://user-images.githubusercontent.com/13227501/193757424-4c577871-1d1a-47f5-9409-663741029f95.mov

**Anything else we need to know?**:

**Relevant WCAG Criteria**:
[2.1.1 Keyboard - Level A](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=335%2C332#keyboard)

**Environment**:
- Grafana version: 9.3.0
- Data source type & version:
- OS Grafana is installed on:
- User OS & Browser:
- Grafana plugins:
- Others:

**Comment by tskarhed** [comment]
FYI: A nice video that describes this issue too is available in [#86833](https://github.com/grafana/grafana/issues/86833)
